Maple2.Database/Storage/Game/GameStorage.cs

```diff
@@ -8,16 +8,15 @@
 namespace Maple2.Database.Storage;
 
 public partial class GameStorage {
-    private readonly DbContextOptions options;
     private readonly ItemMetadataStorage itemMetadata;
     private readonly MapMetadataStorage mapMetadata;
     private readonly AchievementMetadataStorage achievementMetadata;
     private readonly QuestMetadataStorage questMetadata;
     private readonly ILogger logger;
-
+    private readonly DbContextOptions options1;
```

Comment:
rename it back to options

Reply:
oh yea i noticed that happening a lot, thought i got them all guess not